### ISSUE TITLE
/support scripts don't use exec, using unnecessary memory (?)

### ISSUE THREAD

**Issue by wryun** [corbinlc/GNURootDebian#195]
Just wondering why the support scripts (e.g. execInProot/launchInProot) don't use 'exec' and leave busybox sh in memory; I've modified mine and from brief inspection they seem to work fine. Saves a few bytes, and make ps/top look a little prettier.

**Comment by AdamMatthewLTM** [comment]
Hi all,

I am currently assisting the developers in sorting relevant issues for GNURoot Debian on GitHub. If the issue discussed is still relevant, could you please reply to this thread and state the issue needs to be resolved and I will attempt to sort it for you ASAP. If the issue is no longer relevant, please reply to this thread and state it has been resolved, and I will suggest this thread be closed.

Furthermore, if a particular issue is considered to be high priority, please state that you are requesting it to be considered as high priority and I will mark it as such.

Thank you all so much for your time and understanding,
-Adam

**Comment by wryun** [comment]
Yes, it's still relevant - e.g. https://github.com/corbinlc/GNURootDebian/blob/58ae2bb8d173bd39bd6465265de53e1b0788d7fd/GNURootDebianSource/src/armhf/assets/execInProot.replace.mp2

But not high priority.

**Comment by AdamMatthewLTM** [comment]
@wryun ,

My apologies for the delay, and thank you for your response. I am marking this issue as a medium priority request as per your suggestion, and recommend it remain open until further notice. 

The reason why I am marking it as a request is because I personally don't know, and the development team is better suited to answer this question. Once the issues are sorted this thread will be replied to with an answer.

Thank you so much for your patience,
-Adam
